src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -4,18 +4,16 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
-    <!-- replace with $(PackageVersion) once we have a new TFS build -->
-    <RuntimePackageVersion>4.3.0</RuntimePackageVersion>
   </PropertyGroup>
   <ItemGroup>
     <Dependency Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
-      <Version>$(RuntimePackageVersion)-$(ExternalExpectedPrerelease)</Version>
+      <Version>$(PackageVersion)-$(SniExpectedPrerelease)</Version>
```

Comment:
This `PackageVersion` comes from the CoreFX build, right? Is it insync with the version generated by your repo? If this changes in CoreFX repo in future, how will you ensure your repo gets the same version change?

Can we not have another SniPackageVersion that is updated by Maestro and use it here? This way, runtime package versions independent of CoreFX versioning scheme.